en/app.ftl

```diff
@@ -92,6 +92,60 @@ hero-image-copy-unique-html = <strong>Use a unique relay address</strong> for ev
 hero-image-copy-protect-html = … so you can <strong>protect your real email</strong> from tracking and spam.
 hero-image-copy-control-html = Now <em>you’re in control</em> of what’s arriving to your inbox!
 
+# Home Page Version A
+
+# Hero Section
+landing-hero-headline = Protect your real email address to help control your inbox
+landing-hero-body = { -brand-name-firefox-relay } email aliases protect your real email address from public view, automatically forwarding emails to your real inbox. 
+    Now you can receive only the emails you want in your inbox. Sign up with your ⁨{ -brand-name-firefox-account }⁩ to get started.
+
+# How it works section
+landing-how-it-works-headline = How it works
+landing-how-it-works-body = Share { -brand-name-relay } email aliases instead of your real email address to protect your email inbox, as well as your identity.
+landing-how-it-works-step-1-body-cta = Download the { -brand-name-relay } extension for { -brand-name-firefox }
+landing-how-it-works-step-1-body = to generate email aliases automatically wherever you enter your email address online.
```

Comment:
This string and the string should be combined as one string, not separated like this.

Reply:
If you have the screenshot/mockup of this page, that would be helpful with context too.

Reply:
I seperated it into two now.